DAG-insert-elk-sirene.py

```diff
@@ -102,12 +148,12 @@
         dag=dag,
     )
 
-    clean_previous_folder.set_upstream(get_colors)
-    format_sirene_notebook.set_upstream(clean_previous_folder)
-    clean_tmp_folder.set_upstream(format_sirene_notebook)
-    create_elastic_siren.set_upstream(clean_tmp_folder)
-    fill_elastic_siren.set_upstream(create_elastic_siren)
-    check_elastic_index.set_upstream(fill_elastic_siren)
-    update_color_file.set_upstream(check_elastic_index)
-    execute_aio_container.set_upstream(update_color_file)
-    send_email.set_upstream(execute_aio_container)
+    create_sqlite_database << clean_previous_folder << get_colors
+    create_etablissement_table << create_unite_legale_table << create_sqlite_database
+    count_nombre_etablissements_ouverts << count_nombre_etablissements\
+    << create_etablissement_table
+    add_liste_adresses << add_liste_enseignes << count_nombre_etablissements_ouverts
+    create_elastic_index << create_siege_only_table << add_liste_adresses
+    check_elastic_index << fill_elastic_index << create_elastic_index
+    execute_aio_container << update_color_file << check_elastic_index
+    send_email << execute_aio_container
```

Comment:
It's ok but I prefer [this kind of syntax](https://github.com/etalab/dag_inpi_data/blob/56c4264399f3f0f79e9dc6e5f1d562d6f3809640/DAG.py#L312) which is more clear when you have many tasks

Reply:
Oops, I already updated it in the version I didn't push 😅